Assets/Editor/ModelInspector.cs

```diff
@@ -677,22 +886,31 @@ static void ShowOtherInfo(Mesh mesh)
             EditorGUILayout.LabelField("Read/Write Enabled", mesh.isReadable.ToString());
             EditorGUI.indentLevel--;
         }
-
+        
         static void ShowBlendShapeInfo(Mesh mesh)
         {
             var blendShapeCount = mesh.blendShapeCount;
             if (blendShapeCount <= 0)
                 return;
-                
+
             EditorGUILayout.Space();
             EditorGUILayout.LabelField($"Blend Shapes: {blendShapeCount}", EditorStyles.boldLabel);
 
+            var labelHeight = EditorGUIUtility.singleLineHeight;
+
+            var scrollAreaWidth = Screen.width * 0.8f;
+            var scrollAreaHeight = blendShapeCount * labelHeight;
+            if (blendShapeCount > 10)
+                scrollAreaHeight = 10 * labelHeight;
+            
+            m_scrollPos = EditorGUILayout.BeginScrollView(m_scrollPos, GUILayout.Height(scrollAreaWidth), GUILayout.Height(scrollAreaHeight));
```

Comment:
The scrollbar does not look great at small blend shape counts (e.g. one blend shape in one of the meshes in the project).
![Clipboard01](https://user-images.githubusercontent.com/348087/77399946-152bff80-6db3-11ea-949c-a50782f1db6e.png)

Not sure why `GUILayout.Height(scrollAreaWidth)` - height vs width mixup?

How about just doing this instead, i.e. only show scrollbar when blend shape count is "large enough":
```
            var showScroll = blendShapeCount > 10;
            if (showScroll)
                m_scrollPos = EditorGUILayout.BeginScrollView(m_scrollPos, GUILayout.Height(10 * EditorGUIUtility.singleLineHeight));
            EditorGUI.indentLevel++;
            for (int i = 0; i < blendShapeCount; ++i)
            {
                EditorGUILayout.LabelField($"#{i}: {mesh.GetBlendShapeName(i)} ({mesh.GetBlendShapeFrameCount(i)} frames)");
            }
            EditorGUI.indentLevel--;
            if (showScroll)
                EditorGUILayout.EndScrollView();
```